locallayout/store.go

```diff
@@ -165,55 +167,70 @@ func (s *Store) doSave(image v1.Image, withName string) (types.ImageInspect, err
 }
 
 func (s *Store) addImageToTar(tw *tar.Writer, image v1.Image, withName string) error {
-	rawConfigFile, err := image.RawConfigFile()
+	path, err := os.MkdirTemp("", "") // TODO: remove?
 	if err != nil {
 		return err
 	}
-	configHash := fmt.Sprintf("%x", sha256.Sum256(rawConfigFile))
-	if err = addTextToTar(tw, rawConfigFile, configHash+".json"); err != nil {
+	layoutPath, err := layout.Write(path, empty.Index)
```

Comment:
It would be more efficient to add content to the tar as it's being streamed, but that requires more work